src/view/details.js

```diff
@@ -0,0 +1,131 @@
+define(["text!../template/details.html"], function (html) {
+
+    var body = document.body;
+
+    // Load the HTML
+    var template = document.createElement("body");
+    template.innerHTML = html;
+
+    // Itinerary
+    var title = template.querySelector("h2");
```

Comment:
Good use of the native selector API but then you could have avoided the jQuery at all. Why keep jQuery, just for AJAX?